src/Http/HttpResponse.php

```diff
@@ -71,7 +73,7 @@ public static function setCookie($name, $value, $expirySecondsFromNow = 1209600,
         }
 
         if (!Application::current()->unitTesting) {
-            setcookie($name, $value, $expirySecondsFromNow, $path, $domain);
+            setcookie($name, $value, $expirySecondsFromNow, $path, $domain, $secure, $httpOnly);
```

Comment:
@miscampbell could you research to see if the unsetcookie function also needs modified (it may not)

Reply:
@acuthbert it does need to be updated this change has been made also.